server/sendEthernet.go

```diff
@@ -0,0 +1,94 @@
+// Copyright 2018-present the CoreDHCP Authors. All rights reserved
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+
```

Comment:
Even though that' probably the only thing we support today, we should explicitly not build it otherwise
```suggestion

// +build linux
```